bookmarket-website/src/pages/SignUpPage.tsx

```diff
@@ -1,12 +1,84 @@
-import { NavBar } from "@/components/NavBar";
 import "./SignUpPage.css";
 
+import { NavBar } from "@/components/NavBar";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import { useState } from "react";
+
 export function SignUpPage() {
+    const [inputtedFullName, setInputtedFullName] = useState("");
+    const [inputtedEmail, setInputtedEmail] = useState("");
+    const [inputtedPassword, setInputtedPassword] = useState("");
+    const [inputtedConfirmPassword, setInputtedConfirmPassword] = useState("");
+
     return (
-        <div>
+        <div className="signup-page">
             <NavBar />
 
-            <h1>Sign Up</h1>
+            <h1 className="page-title">Sign Up</h1>
+
```

Comment:
Let's add some text here that says, "Already have an account? Click here to log in" where `Click here` is a link to the /login page. It would look something like
```
<p>Already have an account? <Link to="/login">Click here</Link> to log in</p>
```